### PR TITLE
use-case: Add docs for tag-triggered deployment

### PR DESCRIPTION
**Pull request by DamjanBecirovic**

### CODE REVIEW

File: tag-triggered-deployment_5dbacfb104286364bc91238b.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+- A working Semaphore project
+- A basic CI pipeline defined in `.semaphore/semaphore.yml` file. You can use one
+of the documented use cases or language guides as a starting point.
+- A deployment pipeline defined in `.semaphore/deployment.yml` file. You can find
+examples for deployment to AWS, DockerHub, Kubernetes cluster etc. in our
+documentation.
+
+## Set up automatic deployment on a new tag
+
+Let say that you CI pipeline looks something like this:
```
Comment (shiroyasha) [inline tag-triggered-deployment_5dbacfb104286364bc91238b.md]: `your CI` :)

sidenote: run this article through grammarly.

File: tag-triggered-deployment_5dbacfb104286364bc91238b.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    os_image: ubuntu1804
+
+blocks:
+  - name: "Run tests"
+    task:
+      jobs:
+        - name: "Test 1"
+          commands:
+            - checkout
+            - echo "Your commands go here"
```
Comment (shiroyasha) [inline tag-triggered-deployment_5dbacfb104286364bc91238b.md]: Use `make test` it feels more realistic.

File: tag-triggered-deployment_5dbacfb104286364bc91238b.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu1804
+
+blocks:
+  - name: "Run tests"
+    task:
+      jobs:
+        - name: "Test 1"
```
Comment (shiroyasha) [inline tag-triggered-deployment_5dbacfb104286364bc91238b.md]: Just `Tests` is ok.

File: tag-triggered-deployment_5dbacfb104286364bc91238b.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+          commands:
+            - checkout
+            - echo "Your commands go here"
+```
+
+In order to automatically promote this pipeline to deployment phase defined in
+your deployment pipeline you need to expand it with [promotions][promotions] block
+that looks like this:
```
Comment (shiroyasha) [inline tag-triggered-deployment_5dbacfb104286364bc91238b.md]: remove the `that looks like this` part.

File: tag-triggered-deployment_5dbacfb104286364bc91238b.md (hunk 1 above)
Comment (DamjanBecirovic) [reply tag-triggered-deployment_5dbacfb104286364bc91238b.md]: I did that, it did not catch this.

**Pull request merged**